utils/deprecations.py

```diff
@@ -0,0 +1,12 @@
+import logging
+from os.path import basename
+
+log = logging.getLogger(__name__)
```

Comment:
I don't really know how the `logger` works, but it works, and seems to catch the right handler:
`dd-agent status`
> 2015-04-27 18:16:28,589 | WARNING | dd.collector | utils.deprecations(deprecations.py:10) | Using this command is deprecated and will be removed in a future version, for more information see https://github.com/DataDog/dd-agent/wiki/Deprecation-notice--(old-command-line-tools)

`dogstatsd stop`
> 2015-04-27 18:20:43,344 | WARNING | dd.dogstatsd | utils.deprecations(deprecations.py:10) | Using this command is deprecated and will be removed in a future version, for more information see https://github.com/DataDog/dd-agent/wiki/Deprecation-notice--(old-command-line-tools)